.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please remove the ".idea" folder from the repo. And add it to the project `.gitignore`.